mne_bids_pipeline/tests/configs/config_funloc.py

```diff
@@ -0,0 +1,24 @@
+"""Funloc data."""
+
+bids_root = "~/mne_data/MNE-funloc-data"
+deriv_root = "~/mne_data/derivatives/mne-bids-pipeline/MNE-funloc-data"
+task = "funloc"
+ch_types = ["meg", "eeg"]
+
+# Preprocessing
+l_freq = None
+h_freq = 40.0
+use_maxwell_filter: bool = True
+
+
+# Epochs
+epochs_tmin = -0.08
+epochs_tmax = 0.18
+epochs_decim = 10  # 2000->200 Hz
+baseline = (None, 0)
```

Comment:
this is all copied from a different dataset's config, may not be most suitable, TBD.  LMK if you have suggestions here.